com.sublimehq.SublimeText.yaml

```
@@ -0,0 +1,88 @@
+app-id: com.sublimehq.SublimeText
+command: subl
+
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

assuming this supports wayland